src/routes/api/index.ts

```diff
@@ -278,6 +278,18 @@ export function apiRoutes(runner: Runner, client: BitbucketClient, config: Confi
     }),
   );
 
+  router.delete(
+    '/clear-land-when-able-queue',
+    requireAuth('admin'),
+    wrap(async (req, res) => {
+      Logger.verbose('Deleting land-when-able queue', { namespace: 'routes:api:delete-history' });
```

Comment:
namespace: 'routes:api:delete-history' should be namespace: 'routes:api:clear-land-when-able-queue'